### ISSUE TITLE
Unable to set group to the custom created users

### ISSUE THREAD

**Issue by freakster00** [papermerge/papermerge-core#126]
I created groups and then i tried to assign them to my custom created users.After assigning the user with respected group and then saving all the changes,When i try i open that edit user page again I don't see the group getting ticked mark in the edit page.
Is this a backend bug ?

**Comment by freakster00** [comment]
{
  "data": {
    "id": "d104ce00-94ec-4a68-ab14-a8daf5c328ff",
    "attributes": {
      "username": "rohan",
      "email": "rohan@gmail.com",
      "first_name": "rohan",
      "last_name": "poudel",
      "is_active": true,
      "is_staff": true,
      "is_superuser": false,
      "date_joined": "2023-03-11T06:21:47.241940+01:00",
      "perm_codenames": []
    },
    "relationships": {
      "home_folder": {
        "data": {
          "type": "folders",
          "id": "0721d7a5-7560-41d9-bbee-6d66f1d68076"
        }
      },
      "inbox_folder": {
        "data": {
          "type": "folders",
          "id": "f557e317-4ce2-4419-824c-499d5829f526"
        }
      },
      "groups": {
        "data": []
      }
    },
    "type": "users"
  }
}

This seems to be the patch request send from frontend after making necessary changes

**Comment by freakster00** [comment]
![image](https://user-images.githubusercontent.com/121386692/224483622-f70aac13-7c5c-4b58-8dce-5d535f39a037.png)
After submitting these forms ,When i go to edit that user again

**Comment by freakster00** [comment]
@ciur any comment?

**Comment by ciur** [comment]
yes, sounds like a bug. I am not sure on which side it is (backend/frontend) though.

**Comment by ciur** [comment]
@freakster00 Is this bug reproducible on 3.3 ?
